src/components/Payroll/PayrollHistory/PayrollHistory.tsx

```diff
@@ -106,10 +70,19 @@ export const Root = ({ onEvent, companyId, dictionary }: PayrollHistoryProps) =>
     include: ['totals'],
   })
 
+  const { data: wireInRequestsData } = useWireInRequestsListSuspense({
+    companyUuid: companyId,
+  })
+
+  const wireInRequests = wireInRequestsData.wireInRequestList ?? []
+
   const { mutateAsync: cancelPayroll, isPending: isCancelling } = usePayrollsCancelMutation()
 
   const payrollHistory =
-    payrollsData.payrollList?.map(payroll => mapPayrollToHistoryItem(payroll, dateFormatter)) || []
+    payrollsData.payrollList?.map(payroll => ({
+      ...payroll,
+      wireInRequest: wireInRequests.find(wire => wire.paymentUuid === payroll.payrollUuid),
```

Comment:
I'm not in love with merging these two types together. Open to a better clean idea here that doesn't involve us writing a custom type.